FND/Places/NorthAmerica/USPostalServiceAddressesIndividuals.rdf

```diff
@@ -1690,6 +1693,7 @@
 		<rdf:type rdf:resource="&lcc-cr;GeographicRegionIdentifier"/>
```

Comment:
Just because it occupies the same physical space on an address label does not make this a geographic region.